tests/Processor/DateProcessorTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+
+namespace RenanBr\BibTexParser\Test\Processor;
+
+
+use DateTime;
+use DateTimeInterface;
+use DateTimeZone;
+use PHPUnit\Framework\TestCase;
+use RenanBr\BibTexParser\Processor\DateProcessor;
+
+class DateProcessorTest extends TestCase
+{
+    public function testDateYearAndMonth()
+    {
+        $processor = new DateProcessor();
+        $entry = $processor([
+            'month' => '"1~" # jan',
+            'year' => '2000',
+        ]);
+        $this->assertEquals('2000', $entry['year']);
+        $this->assertEquals('1~jan', $entry['month']);
+        /** @var DateTime $dateTime */
+        $dateTime = $entry['_date'];
+        $this->assertEquals('2000-01-01T00:00:00+00:00', $dateTime->format(DateTimeInterface::ATOM));
```

Comment:
Use `DateTime::ATOM` instead

This constant is placed into `DateTime` class in PHP < 7.2 (see [this](https://3v4l.org/OSM84) and [that](https://3v4l.org/QfAmp)) and, moreover, we must keep compatibility with [php >=5.6.0](https://github.com/renanbr/bibtex-parser/blob/b1922ca1654ce7fffe5f9648099e06ef17994f9b/composer.json#L22)